src/utils/transaction/prepareRequest.ts

```diff
@@ -43,7 +42,7 @@ export type PrepareRequestReturnType<
   nonce: SendTransactionParameters['nonce']
 }
 
-export const defaultTip = parseGwei('1.5')
+export const defaultTip = 1500000000n // 1.5 gwei
```

Comment:
Adding some numeric separators

```suggestion
export const defaultTip = 1_500_000_000n // 1.5 gwei
```